.github/workflows/tests.yaml

```diff
@@ -133,13 +133,26 @@ jobs:
           clang++ --version
 
       # Extract our access key for our build cache.
-      - name: Extract access key
-        if: steps.filter.outputs.ignore == 'false'
+      - name: Extract access key (read-only)
```

Comment:
Maybe rename?

```suggestion
      - name: Set up read-only cache access
```

Reply:
Going with "Extract access key" -> "Set up remote cache", keeping parenthetical for read-write vs read-only